pkgs/development/libraries/science/math/cutensor/default.nix

```
@@ -33,5 +33,13 @@ rec {
     cudatoolkit = cudatoolkit_11_2;
   };
 
-  cutensor_cudatoolkit_11 = cutensor_cudatoolkit_11_2;
+  cutensor_cudatoolkit_11_3 = cutensor_cudatoolkit_11_0.override {
+    cudatoolkit = cudatoolkit_11_3;
+  };
+
+  cutensor_cudatoolkit_11_4 = cutensor_cudatoolkit_11_0.override {
+    cudatoolkit = cudatoolkit_11_4;
+  };
+
+  cutensor_cudatoolkit_11 = cutensor_cudatoolkit_11_4;
```

```suggestion
  cutensor_cudatoolkit_11 = cutensor_cudatoolkit_11_2;
```
I just discovered that this would bloat the closure of referrers which also use cudnn_cudatoolkit_11, such as cupy.
Let's plan on a big flag day bump for cudnn/cutensor/cudatoolkit -> 11.4.
Exception being the samples, which I think are fine to bump here since nothing depends on them.

---

Ah good catch, applied your suggestion!